### DISCUSSION TITLE
Anonymous auth

### DISCUSSION THREAD

**Discussion by dqii** [supabase/supabase#2945] [Questions] [answered]
I wanted to ask if there were plans to support anonymous auth anytime soon?

https://github.com/supabase/gotrue/issues/68
I saw here that it's possible with a fake email hack but it seemed like this would not work if I wanted to support the user making an account via a different OAuth provider (phone, Facebook) later.

**Comment by kangmingtay** [comment, marked as answer]
Hey @dqii, we don't have plans to support anonymous auth anytime soon. We're currently prioritising security & stability but we'll definitely consider it next! 

> I saw here that it's possible with a fake email hack but it seemed like this would not work if I wanted to support the user making an account via a different OAuth provider (phone, Facebook) later.

Yeah this wouldn't work right now but we are currently refactoring gotrue so that we can implement some account linking features to make this hack possible.

**Reply by juliuscc** [reply]
Does there exist some issue that we can follow or help with?
